sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -228,7 +228,12 @@ async def create_container(
         response_hook = kwargs.pop('response_hook', None)
         offer_throughput = kwargs.pop('offer_throughput', None)
         if offer_throughput is not None:
-            request_options["offerThroughput"] = offer_throughput
+            if isinstance(offer_throughput, int):
+                request_options["offerThroughput"] = offer_throughput
+
+            if isinstance(offer_throughput, object):
```

Comment:
I think we need to work on this logic a bit. Maybe something like:
```
try:
    if offer.auto_scale_max_throughput or offer.auto_scale_increment_percent:
        request_options['autoUpgradePolicy'] = stringify_auto_scale(offer=offer_throughput)
    elif offer.offer_throughput:
        request_options["offerThroughput"] = offer.offer_throughput
    else:
        raise ValueError(...)
except AttributeError:
    if isinstance(offer, int):
        request_options["offerThroughput"] = offer_throughput
    else:
        ...
```

Reply:
I tried what you have recommended but it doesn't really work out the same way we would want it to especially because offer doesn't exist here and I tried to use the ThroughputProperties instead but that doesn't work out either, I understand technically that is true but it is not causing any issues with wanting to set a offer throughput or setting the autoscale settings.

Reply:
nvm just needed to change 'offer' to offer_throughput and it worked thank you!